acceptance/tests/ticket_6928_puppet_master_parse_fails.rb

```diff
@@ -11,7 +11,7 @@
 
 step "Master: use --parseonly on an invalid manifest, should return 1 and issue deprecation warning"
 on master, puppet_master( %w{--parseonly /tmp/bad.pp} ), :acceptable_exit_codes => [ 1 ]
-  assert_match(/--parseonly has been removed. Please use \'puppet parser validate <manifest>\'/, stdout, "Deprecation warning not issued for --parseonly on #{master}" ) unless agent['locale'] == 'ja'
+  assert_match(/--parseonly has been removed. Please use \'puppet parser validate <manifest>\'/, stdout, "Deprecation warning not issued for --parseonly on #{master}" ) unless master['locale'] == 'ja'
```

Comment:
This failed on a Japanese agent, so this string in the stdout here must be coming from the agent. I don't think confining when master is in Japanese will work (we're also not testing Japanese masters at the moment).